liquid-compiler/src/filter.rs

```diff
@@ -1,101 +1,121 @@
-use std::fmt::{self, Debug};
-
-use liquid_error;
+use super::FilterArguments;
+use liquid_error::Result;
+use liquid_interpreter::Context;
 use liquid_value::Value;
+use std::fmt::Debug;
```

Comment:
Don't remember if this is the recommended style but I split `use`s up into three blocks

```
use std

use external

use internal
```